maps_red/ticketing-bundle/0.2/config/packages/ticketing.yaml

```diff
@@ -0,0 +1,58 @@
+doctrine:
+    orm:
+        resolve_target_entities:
+            Maps_red\TicketingBundle\Model\TicketInterface: App\Entity\Ticket
+            Maps_red\TicketingBundle\Model\TicketCategoryInterface: App\Entity\TicketCategory
+            Maps_red\TicketingBundle\Model\TicketCommentInterface: App\Entity\TicketComment
+            Maps_red\TicketingBundle\Model\TicketHistoryInterface: App\Entity\TicketHistory
+            Maps_red\TicketingBundle\Model\TicketKeywordInterface: App\Entity\TicketKeyword
+            Maps_red\TicketingBundle\Model\TicketStatusInterface: App\Entity\TicketStatus
+            Maps_red\TicketingBundle\Model\TicketPriorityInterface: App\Entity\TicketPriority
+            Symfony\Component\Security\Core\User\UserInterface: App\Entity\User
+ticketing:
+
+    # The name of the default status
+    default_status_name:  open
+
+    # Enable the seen/unseen display for tickets
+    enable_history:       true
+
+    # Set a role that will have access to the "staff" tickets
+    restricted_tickets_role: ROLE_USER
+
+    # if true, all new tickets will only be accessible to the owner and the restricted_ticket_role. If false, all new tickets will be public
+    enable_ticket_restriction: true
+    entities:
+        ticket:               App\Entity\Ticket
```

Comment:
These entities does not exists. Maybe you should provide an implementation of some basic entities?

Reply:
These entities are supposed to be created by the ``make:ticketing`` command from the bundle (in the manifest under composer-scripts) 
But is it better to initiate them directly in the recipe ?

Reply:
Im not sure, you are way more knowledgeable about the bundle. 
But `make:ticketing` will run every time you do a composer update ([docs](https://github.com/symfony/recipes#composer-scripts-configurator)). Is that really want you want? 

By using the command to generate entities you get more control over the output. That is up to you. Just make sure the bundle does not complain if these entites does not exist.

Reply:
Ah indeed, thanks for the review, I updated the recipe